unit-13/main.js

```diff
@@ -0,0 +1,69 @@
+// Exercise 1:
+let date = new Date()
+const day = date.getDate()
+const month = date.getMonth() + 1
+const year = date.getFullYear()
+const hours = date.getHours()
+const minutes = date.getMinutes()
+const seconds = date.getSeconds()
+const currentTime = `${day}-${month}-${year} ${hours}:${minutes}:${seconds}`
+
+const getTimeOut = () => {
+  console.log(`Hover lúc: ${currentTime}`)
+};
```

Comment:
Xóa dấu chấm phẩy phía cuối đi
```suggestion
}
```
Đây ko phải là object, mà đây là khai báo 1 hàm, ko có khai báo 1 hàm nào mà có dấu chấm phẩy ở cuối cả